### DISCUSSION TITLE
"Americana" stack for US Federal web sites?

### DISCUSSION THREAD

**Discussion by lpsinger** [remix-run/remix#7068] [Proposals]
We have been using Remix at NASA: we have an open data site for astronomers called General Coordinates Network (GCN, https://gcn.nasa.gov). There are a couple of things that we have added on top of Remix for building a US Federal web site: the [US Web Design System (USWDS)](https://designsystem.digital.gov) for styling, the [react-uswds](https://github.com/trussworks/react-uswds) React component kit, and some hosting defaults that make sense at least at NASA.

We are preparing to run a small boot camp at NASA on modern web development for scientists, and it occurs to us that we should package up our Remix + USWDS tech stack as a template. Would Remix be interested in us contributing this as an official [Remix stack](http://remix.run/docs/en/main/pages/stacks)?

I already thought of a name for it: Americana.

**Comment by sergiodxa** [comment]
AFAIK Remix only provides three stacks and the idea is that individuals and organizations can create their own stacks (private or public) and publish them to GitHub.

I think you should publish it under the NASA organization (or any other US government organization on GitHub) and then share it with the Remix community!

**Reply by MichaelDeBoey** [reply]
We also recommend adding the [`remix-stack`](https://github.com/topics/remix-stack) tag to the GitHub repo for better visibility by the community

**Reply by lpsinger** [reply]
Here's a very rough first draft: https://github.com/nasa-gcn/americana-stack
